src/azure-cli/azure/cli/command_modules/appconfig/_snapshot_custom_client.py

```diff
@@ -407,9 +407,11 @@ def list_snapshots(
 
         _headers = kwargs.pop("headers", {}) or {}
 
+        status_filter = ",".join(status) if status else None
+
         initial_request = build_list_snapshots_request(
             name_filter=name,
-            status_filter=status,
+            status_filter=status_filter,
```

Comment:
Does passing `None` have the same effect as passing `*` as status filter?

Reply:
I think it may cause confusion if we support * sign here. In all other filters when we support *, we also support prefix filtering. But that's not the case with status filter. So I think it's better to omit the status instead of providing * as an option. We could add more details to the help message though - something like:
`Filter snapshots by their status. If no status specified, return all snapshots by default. `

Reply:
Sounds good! This has been updated, thanks!